UIKitDZ/UIKitDZ/Controllers/BillViewController.swift

```diff
@@ -0,0 +1,137 @@
+//
+//  BillViewController.swift
+//  UIKitDZ
+//
+//  Created by 2lup on 28.09.2022.
+//
+
+import UIKit
+
+/// BillViewController: Экран оплаты заказа
+final class BillViewController: UIViewController {
+    
+    private lazy var descriptionOrederLabel: UILabel = {
+        $0.font = UIFont(name: "Verdana", size: 28)
+        $0.text = """
+            Ваш заказ:
+            1. Пицца "\(currentPizza.namePizza)"
+            """
+        $0.numberOfLines = 0
+        $0.textAlignment = .left
+        $0.textColor = .black
+        return $0
+    }(UILabel())
+    
+    private lazy var paymentInCashLabel: UILabel = {
+        $0.text = "Оплата наличными"
+        $0.textAlignment = .left
+        $0.font = UIFont(name: "Verdana", size: 20)
+        $0.textColor = .black
+        return $0
+    }(UILabel())
+    
+    private lazy var paymentInCashSwitch: UISwitch = {
+        $0.setOn(false, animated: false)
+        $0.tag = 0
+        $0.addTarget(self, action: #selector(changePaymentAction(target:)), for: .valueChanged)
+        return $0
+    }(UISwitch())
+    
+    private lazy var paymentOnlineLabel: UILabel = {
+        $0.text = "Оплата картой"
+        $0.textAlignment = .left
+        $0.font = UIFont(name: "Verdana", size: 20)
+        $0.textColor = .black
+        return $0
+    }(UILabel())
+    
+    private lazy var paymentOnlineSwitch: UISwitch = {
+        $0.setOn(false, animated: false)
+        $0.tag = 1
+        $0.addTarget(self, action: #selector(changePaymentAction(target:)), for: .valueChanged)
+        return $0
+    }(UISwitch())
+    
+    private lazy var payButton: UIButton = {
+        $0.setTitle("Pay", for: .normal)
+        $0.titleLabel?.font = UIFont(name: "Verdana", size: 20)
+        $0.backgroundColor = .black
+        $0.tintColor = .white
+        $0.addTarget(self, action: #selector(payButtonAction), for: .touchUpInside)
+        return $0
+    }(UIButton())
+    
+    var currentPizza = DataPizza()
+    var availableCash = false
```

Comment:
```suggestion
    var isAvailableCash = false
```
is перед бужом всегда